src/components/CommentList/CommentList.scss

```diff
@@ -1 +1,7 @@
-//
```

Comment:
Use line comments instead of block comments. Replace '//' with '// CommentList styles' to provide more context.